PocketKit/Sources/PocketKit/Home/HomeViewController.swift

```diff
@@ -168,13 +174,23 @@ extension HomeViewController {
             cell.configure(model: viewModel)
             return cell
         case .recommendationHero(let objectID):
-            let cell: RecommendationCell = collectionView.dequeueCell(for: indexPath)
-            guard let viewModel = model.recommendationHeroViewModel(for: objectID,  at: indexPath) else {
+            if traitCollection.userInterfaceIdiom == .pad && traitCollection.horizontalSizeClass == .regular {
```

Comment:
Good call. I added a method to the layout section provider since I think extending `UITraitCollection` could lead to collisions if other parts of the app need similar logic. 1cbb803adc0a4aacf7e4bfaea302e0ae57291378